src/commands/registries/azure/tasks/scheduleRunRequest.ts

```diff
@@ -24,7 +24,12 @@ import { getStorageBlob } from '../../../../utils/lazyPackages';
 const idPrecision = 6;
 const vcsIgnoreList = ['.git', '.gitignore', '.bzr', 'bzrignore', '.hg', '.hgignore', '.svn'];
 
-export async function scheduleRunRequest(context: IActionContext, requestType: 'DockerBuildRequest' | 'FileTaskRunRequest', uri: vscode.Uri | undefined): Promise<() => Promise<AcrRun>> {
+export enum RootStrategy {
```

Comment:
I'd make this enum explicit in its typing instead of the implicit default integer enum. Personally I prefer strings because when debugging, having variables with values `0` or `1` just mean I have to go look at the source code to find out what they really mean. You can make it explicitly string like so:

```typescript
export enum RootStrategy {
    Default = 'Default',
    DockerfileFolder = 'DockerfileFolder',
}
```

An added benefit is that there is zero ambiguity about whether the AKS DevX extension should pass an integer or string.